src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,75 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    ///////////////////////////////////////
+    ///////////// Read from file //////////
+    ///////////////////////////////////////
     public void getStatistic(String fromFileName, String toFileName) {
+        // Declare a File object to represent the file
+        File fromFile = new File(fromFileName);
+        // Initialize a variable to store the total supply and total buy
+        int supply = 0;
+        int buy = 0;
+        try {
+            // Create a BufferedReader to read the file
+            BufferedReader reader = new BufferedReader(new FileReader(fromFileName));
+            // Create a StringBuilder to store the contents of the file
+            StringBuilder stringBuilder = new StringBuilder();
+            // Read the first line of the file
+            String value = reader.readLine();
+            // Create a Temp String array with size 2(buy,2)
+            String[] temp = new String[2];
+            // Continue reading lines until there are no more lines to read
+            while (value != null) {
+                // Append the current line to the StringBuilder
+                stringBuilder.append(value);
+                //Checking if line contains buy, if TRUE store to temp array value of line
+                // Example = Line: (buy,4) = buy[0]arr, 4[1]arr
+                if (value.contains("buy")) {
+                    temp = value.split(",");
+                    // addition every line with "buy" to var - buy
+                    buy += Integer.parseInt(temp[1]);
```

Comment:
```suggestion
                    buy += Integer.parseInt(value.split(",")[1]);
```
temp variable is not needed, it is better to remove it